sites/footers/reports/a11y-1.json

```
@@ -0,0 +1,905 @@
+{
+  "@context": "https://wet-boew.github.io/GCWeb/sites/footers/index.json-ld",
+
+  "@id": "https://wet-boew.github.io/GCWeb/sites/footers/reports/a11y-1.json",
+  "id": "1",
+  "@type": "earl:Assertion",
+  "earl:test": ["WCAG20", "WCAG21", "WCAG22"],
+
+  "subjectUrl": "https://universallabs.org/wet/labs/pr2043/sites/footers/footers-en.html",
```

I looked and this snapshot is what the site footer was on November 22, 2022 9:22AM which should match of the content merged via the PR 2043.

And based on the change history, that page only modified on December 2022.

Let use the published URL from our GCWeb website instead which are matching the same content of the UAT site published on universallabs.org

```suggestion
  "subjectUrl": "https://wet-boew.github.io/GCWeb/sites/footers/footers-en.html",
```

---

Let's keep it as is considering that was the actual tested page at that time. We could probably add a reference to the wet-boew page also considering it is equivalent.